src/components/Calculator.js

```diff
@@ -7,13 +7,16 @@ import Buttons from "./Buttons";
 
 const Calculator = () => {
   const [value, setValue] = React.useState(0)
+
   const operators = ["X", "%", "±", "AC", "+", "-", "/", "="]
 
   const onButtonClick = (val) => {
     if (!operators.includes(val)) {
-      setValue(val)
-    }
+      setValue((prevValue) => {
+        return parseInt(prevValue.toString() + val)
```

Comment:
You're better off using a string interpolation to prevent any possible undesirable effects from using the `+`.


```
parseInt(`${prevValue}${val}`)
```